Initial Installer Commit

calr0x opened the pull request:
Fixes #

This is the installer exactly as designed. It has been modified slightly as it will now reside within the ot-node directory as opposed to a separate repo which then clones ot-node.

I expect some suggestions to change the wording of various components. Just let me know what I can do!

To install:

1. Login new server.
2. Get the GraphDB zip file in /root
3. git clone https://github.com/OriginTrail/ot-node.git && cd ot-node && git checkout v6/release/testnet && installer/installer.sh

kotlarmilos committed:
first commit

NZT48 committed:
Fix

kotlarmilos committed:
version bump

kotlarmilos committed:
update config

kotlarmilos committed:
update readme

kotlarmilos committed:
version bump

kotlarmilos committed:
Merge pull request #1656 from OriginTrail/v6/develop

V6/develop

NZT48 committed:
Update noderc example

NZT48 committed:
Merge pull request #1657 from OriginTrail/v6/feature/config-example

Update noderc example

kotlarmilos committed:
Merge pull request #1658 from OriginTrail/v6/develop

Update noderc example

kotlarmilos committed:
Update README.md

kotlarmilos committed:
update logs

kotlarmilos committed:
bump version

kotlarmilos committed:
Merge pull request #1659 from OriginTrail/v6/develop

V6/develop

NZT48 committed:
Upload postman collection

branarakic committed:
Update README.md

NZT48 committed:
Merge branch 'v6/develop' of https://github.com/OriginTrail/ot-node into v6/develop

kotlarmilos committed:
remove logs

kotlarmilos committed:
Merge branch 'v6/develop' of https://github.com/OriginTrail/ot-node into v6/develop

kotlarmilos committed:
update

kotlarmilos committed:
Merge pull request #1660 from OriginTrail/v6/develop

V6/develop

kotlarmilos committed:
buffers fix

kotlarmilos committed:
Merge pull request #1661 from OriginTrail/v6/develop

buffers fix

kotlarmilos committed:
Update streams handling and add telemetry hub measuring

kotlarmilos committed:
refactor data service and make uniform streams response

kotlarmilos committed:
refactor runQuery in data service

kotlarmilos committed:
Merge pull request #1662 from OriginTrail/v6/develop

Update streams handling and add telemetry hub measuring

calr0x committed:
.

calr0x committed:
.

calr0x committed:
.

calr0x committed:
.

calr0x committed:
.

calr0x committed:
.

calr0x committed:
.

calr0x committed:
.

calr0x committed:
Update installer.sh

NZT48 commented on installer/README.md:
````diff
@@ -0,0 +1,20 @@
+3. Execute the following command:
+
+```
+apt install git -y && git clone https://github.com/calr0x/OT-KwallaBetaInstalla.git && cd OT-KwallaBetaInstalla && ./installer.sh
````
Update this command to work with new directory name (installer)

calr0x replied:
Keep them coming!:)

NZT48 commented on installer/installer.sh:
```diff
@@ -0,0 +1,389 @@
+
+echo "Installing the latest OriginTrail V6 Beta!"
+echo "This is a community installer. For support please visit the Communtiy Node Support Telegram group at https://t.me/otnodegroup."
+echo "Origintrail will NOT be able to support this installer."
```
This lines can be removed :)

calr0x committed:
Remove disclaimer and update install directions

calr0x committed:
Add additional verbiage for installation

calr0x committed:
Update language to remove "my" and other language fixes

Changed "my" to "the script"
Cleaned up the language to be more formal and concise.

calr0x committed:
Remove SSL mention

calr0x commented:
Thought I would direct attention that I am running the GraphDB as a system service in the installer. I feel unification in the end-result between the official instructions and the installer is important for future support in that if someone needs help in Discord/TG we can make certain assumptions about how to restart graphdb (nohup vs systemctl restart graphdb).

Do you have any thoughts on this topic? We can convert the node from forever to systemctl or remove graphdb as a service and use nohup, etc.

My advice is that the manual instructions and the installer should use the same methods for consistency...

Let me know!

NZT48 commented:
@calr0x I agree with you, but I think that right now it would be useful to make exception because of the feature that GraphDB service can do auto restart on failure. In the future we can try to make everything consistent, and I would try to adapt manual instructions to be compatibile with the script.

PS I tested the script and it was much better user experience then installing manually, really awesome work!

calr0x commented:
That's how I felt too and why I did it... :)

Just wanted to make sure and point it out!

Thx, NZT!

NZT48 commented:
@calr0x  I made a mistake, wanted to change to merge into v6/develop and by accident clicked on develop. Will fix this but need Brana's help as he is the code owner. Anyway, everything is great and will be merged into v6/develop soon, and after that I will comment here when I have made changes to official documentation

calr0x commented:
Lol np!

NZT48 commented:
Also @calr0x can you copy all this changes to v6/develop on you repo, thanks !

calr0x commented:
> Also @calr0x can you copy all this changes to v6/develop on you repo, thanks !

Will do!
